google/cloud/ndb/_datastore_api.py

```diff
@@ -89,6 +89,7 @@ def rpc_call():
         rpc = _remote.RemoteCall(call, rpc_name)
         utils.logging_debug(log, rpc)
         utils.logging_debug(log, "timeout={}", timeout)
+        utils.logging_debug(log, request)
```

Comment:
My only concern here is potential overhead -- the `request` protobuf message could be arbitrarily large, especially for write operations.  If that isn't a concern (becasue `utils.logging_debug` does a short-curcuit), then this should be fine.

Reply:
`utils.logging_debug` does nothing unless the `NDB_DEBUG` environment variable is set to something truthy.